.ci/wercker.sh

```diff
@@ -182,7 +189,7 @@ no-error-contribution)
      -Dcheckstyle.configLocation=../../../config/checkstyle_checks.xml
   cd ../
   cd ../../
-  rm -rf checkstyle
+  removeFolderWithProtectedFiles checkstyle
```

Comment:
This should be `contribution`

Reply:
Fixed, but wercker is failing.  Could you please share the log with me?

Reply:
Now it fails with `find: ‘blog-sonar-build-breaker’: No such file or directory`

Reply:
Fixed.